content/developers/internals/data-model/usage-example.en.md

```diff
@@ -0,0 +1,288 @@
+
+### Routes
+
+A `Route` in OSRD represents an itinerary in the infrastructure. A train's path can be composed of several `Route` objects. They are used when interacting with path reservation for trains and are thus part of the interlocking system. For more information on routing and reservations see the dedicated pages ([link](/fr/developers/internals/simulation-model/)).
+
+It is represented with the following attributes:
+
+* `entry_point` and `exit_point`: in *OSRD* those are generally references to detectors or buffer stops.
+* `path`: The list of track section ranges a train can pass on to go from entry to exit. There can be two Routes with the same entry and exit points and two different paths.
+* `release_detectors`: The list of detectors that can be released after the train has crossed them. This allows for "hard" reservation when the list is empty and "soft" when the list contains all the detectors seen along the way.
+
+In our infrastructure, all possible routes were generated automatically by the script.
+
+## Signaling
+
+Thanks to interlocking, trains are located and allowed to move. It's a good start, but meaningless until trains are made aware of it. This is where the `Signal`s come into play: signals react to interlocking, and can be seen by trains.
+
+How trains react to signals depends on signal type.
+
+Here are the most important attributes for the signals:
+
+* `linked_detector`: The detector it is linked to.
+* `type_code`: The type of signal.
+* `direction`: The direction it protects, which can simply be interpreted as the way in which it can be seen by an incoming train (since there are lights only on one side...).
+* Cosmetic attributes like `angle_geo` or `side` which control the way in which the signals are displayed in the front-end.
+
+Here is a visualization of how one can represent a signal, and which direction it protects.
+
+![Signal direction example](../svg_diagrams/signal_dir.en.svg)
+
+<br />
+
+The way the signals are arranged in an infrastructure depends on the local convention and can vary a lot from region to region.
+
+Here is the convention we adopted in our example infrastructure:
+
+* We add two signals for each detector that are simply cutting the track sections into smaller TVDS, one in each direction.
+* For detectors marking the entry of switches, we add only one signal, visible in the direction from which you enter the switch. Indeed, we chose not to be able to stop trains inside switches, because a train stopped in a switch makes it unusable by other trains.
+* For the detectors that are in between two switches, we do not add signals, for the same reason as before.
+
+And there we have all the signals placed. Please note that we do not represent detectors that are linked to at least one signal.
```

Comment:
```suggestion
Note that detectors linked to at least one signal are not represented, as there are not signals without associated detectors in this example.
```

Is my proposed change correct? It's not always the case in real life

Reply:
It is